test/controllers/respondent_controller_test.exs

```diff
@@ -40,7 +40,14 @@ defmodule Ask.RespondentControllerTest do
                                                          "value" => response.value,
                                                          "name" => response.field_name
                                                        }
-                                                     ]
+                                                     ],
+                                                     "stats" => %{
```

Comment:
can't a constant `empty_stats` be used to avoid the duplication to make test pass?